integration-tests/src/test/groovy/com/okta/sdk/tests/it/EventHooksIT.groovy

```diff
@@ -0,0 +1,254 @@
+import static org.hamcrest.Matchers.hasSize
+import static org.hamcrest.Matchers.notNullValue
+
+/**
+ * Tests for /api/v1/eventHooks
+ * @since 2.0.0
+ */
+class EventHooksIT extends ITSupport {
+
+    @Test
+    void createEventHookTest() {
+        String name = "event-hook-java-sdk-${UUID.randomUUID().toString()}"
+
+        EventHook createdEventHook = client.instantiate(EventHook)
+            .setName(name)
+            .setEvents(client.instantiate(EventSubscriptions)
+                .setType(EventSubscriptions.TypeEnum.EVENT_TYPE)
+                .setItems(["user.lifecycle.create", "user.lifecycle.activate"]))
+            .setChannel(client.instantiate(EventHookChannel)
+                .setType(EventHookChannel.TypeEnum.HTTP)
+                .setVersion("1.0.0")
+                .setConfig(client.instantiate(EventHookChannelConfig)
+                    .setUri("https://www.example.com/eventHooks")
+                    .setHeaders([ client.instantiate(EventHookChannelConfigHeader)
+                                      .setKey("X-Test-Header")
+                                      .setValue("Test header value")])
+                    .setAuthScheme(client.instantiate(EventHookChannelConfigAuthScheme)
+                .setType(EventHookChannelConfigAuthSchemeType.HEADER)
+                .setKey("Authorization")
+                .setValue("Test-Api-Key"))))
+        registerForCleanup(createdEventHook)
+
+        assertThat(createdEventHook.getId(), notNullValue())
+        assertThat(createdEventHook.getName(), equalTo(name))
+        assertThat(createdEventHook.getEvents(), hasSize(2))
+        assertThat(createdEventHook.getChannel().getConfig().getUri(), equalTo("https://www.example.com/eventHooks"))
+    }
+
+    @Test
+    void getEventHookTest() {
+        String name = "event-hook-java-sdk-${UUID.randomUUID().toString()}"
+
+        EventHook createdEventHook = client.instantiate(EventHook)
+            .setName(name)
+            .setEvents(client.instantiate(EventSubscriptions)
+                .setType(EventSubscriptions.TypeEnum.EVENT_TYPE)
+                .setItems(["user.lifecycle.create", "user.lifecycle.activate"]))
+            .setChannel(client.instantiate(EventHookChannel)
+                .setType(EventHookChannel.TypeEnum.HTTP)
+                .setVersion("1.0.0")
+                .setConfig(client.instantiate(EventHookChannelConfig)
+                    .setUri("https://www.example.com/eventHooks")
+                    .setHeaders([ client.instantiate(EventHookChannelConfigHeader)
+                                      .setKey("X-Test-Header")
+                                      .setValue("Test header value")])
+                    .setAuthScheme(client.instantiate(EventHookChannelConfigAuthScheme)
+                        .setType(EventHookChannelConfigAuthSchemeType.HEADER)
+                        .setKey("Authorization")
+                        .setValue("Test-Api-Key"))))
```

Comment:
See note about builder above (IIRC async hooks are similar, except they need the subscription list).

Reply:
noted.